srcpkgs/buku/patches/use-system-cacerts.patch

```diff
@@ -0,0 +1,80 @@
+From d294a09e802694f7cf1fbf64e99818759d934b34 Mon Sep 17 00:00:00 2001
+From: Rasmus Thomsen <rasmus.thomsen@protonmail.com>
+Date: Sat, 17 Nov 2018 14:45:08 +0100
+Subject: [PATCH] use-system-cacerts.patch
+
+---
+ buku.py  | 15 ++++++++-------
+ setup.py |  1 -
+ 2 files changed, 8 insertions(+), 8 deletions(-)
+
+diff --git a/buku.py b/buku.py
+index dc64850..0e515de 100755
+--- buku.py
++++ buku.py
+@@ -19,7 +19,8 @@
+ 
+ import argparse
+ from bs4 import BeautifulSoup
+-import certifi
++def where():
++    return '/etc/ssl/certs/ca-certificates.crt'
```

Comment:
Move the `where` function after the various `import`, just above the `class` definition.